src/tree/SubscriptionTreeItem.ts

```diff
@@ -58,6 +58,7 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         const executeSteps: AzureWizardExecuteStep<IStaticWebAppWizardContext>[] = [];
 
         if (!context.advancedCreation) {
+            wizardContext.sku = SkuListStep.getSkus()[0];
```

Comment:
Fwiw, sku is another thing we decided was worth including in basic create for App Service.